src/commands/init/index.ts

```diff
@@ -544,7 +553,7 @@ async function detectTests(pathToExistingProject: string): Promise<string | unde
 }
 
 async function readRootCargoToml(pathToProject: string) {
-  const rootTomlPath = path.resolve(pathToProject, "Cargo.toml");
+  const rootTomlPath = path.resolve(pathToProject, "Cargo.toml.hbs.hbs");
```

Comment:
Why did you add the ".hbs.hbs" extension in the convert section of the `init` command? @prxgr4mm3r

Reply:
I think my IDE did it automatically for some reason. Already fixed it.